WordPress/WordPressTest/PostCardCellTests.swift

```diff
@@ -490,13 +490,22 @@ class PostCardCellTests: XCTestCase {
         expect(self.postCell.statusLabel.textColor).to(equal(UIColor.warning))
     }
 
-    func testShowsUnsavedChangesMessageWhenPostHasAutosave() {
+    func testShowsVersionConflictMessageWhenPostHasRemoteAutosave() {
         let post = PostBuilder(context).with(remoteStatus: .sync).autosaved().build()
 
         postCell.configure(with: post)
 
-        expect(self.postCell.statusLabel.text).to(equal(i18n("You've made unsaved changes to this post")))
-        expect(self.postCell.statusLabel.textColor).to(equal(UIColor.warning(.shade40)))
+        expect(self.postCell.statusLabel.text).to(equal(i18n("Version Conflict")))
```

Comment:
This test checks that the local autosave conflict label has replaced the remote autosave conflict label for remote conflicts. This was back when we intended to use the same dialog for both local and remote conflicts which after seeing that they exhibit very different behavior and speaking with yourself we decided against.

Perhaps it does make sense to have different labels for remote and local autosave conflicts now that we're using two distinct dialogs. What do you think? It will be easy enough to revert this commit and go back to having the old style label for remote auto saves conflicts.

#### Current Remote autosave conflict label
![image](https://user-images.githubusercontent.com/18557720/95489912-5ff81800-098f-11eb-887a-d554832aad1d.png)

#### New local autosave conflict label
![image](https://user-images.githubusercontent.com/18557720/95490442-24aa1900-0990-11eb-93fe-43a00639860a.png)

Cliffs: Do we want to keep different labels for these both?

Reply:
As chatted via DM, I think the previous label was better for autosave conflicts.  We can re-evaluate the UI once we're finished though.